charts/karmada/templates/_karmada_webhook_configuration.tpl

```diff
@@ -51,20 +51,6 @@ webhooks:
     sideEffects: None
     admissionReviewVersions: ["v1"]
     timeoutSeconds: 3
-  - name: clusteroverridepolicy.karmada.io
-    rules:
-      - operations: ["CREATE", "UPDATE"]
-        apiGroups: ["policy.karmada.io"]
-        apiVersions: ["*"]
-        resources: ["clusteroverridepolicies"]
-        scope: "Cluster"
-    clientConfig:
-      url: https://{{ $name }}.{{ $namespace }}.svc:443/validate-clusteroverridepolicy
```

Comment:
Why remove this? We have this webhook endpoint.

https://github.com/karmada-io/karmada/blob/e237614954a64b2595b9ec18128834a538975518/cmd/webhook/app/webhook.go#L122

Reply:
it's a validate admission, should moved to ValidatingWebhookConfiguration

Reply:
Great, nice finding!